config/action.d/allow-iptables-multiport.conf

```diff
@@ -0,0 +1,59 @@
+# Fail2Ban configuration file for allowing hosts
+#
+# WARNING
+# Please be aware that all users behind NAT will access the service on the specified port.
+# You should protect this service with another jail that has very long bantime.
+
+[INCLUDES]
+
+before = iptables-common.conf
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = iptables -N f2b-<name>
+              iptables -A f2b-<name> -j <blocktype>
```

Comment:
Does `RETURN` work?

Thinking out loud:

User gets a 200 from apache, this jail adds the user to the f2b-pass2allow chain for ftp.
User opens a connection to ftp, INPUT > f2b-pass2allow -j RETURN > ftp REJECT.  Assuming of course your INPUT looks like INPUT > ftp > REJECT or falls defaults to a REJECT/DROP.